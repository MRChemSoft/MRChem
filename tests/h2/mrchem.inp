# vim:syntax=sh:

rel_prec = 1.0e-4                 # Overall relative precision
printlevel = 0

MRA {
  order = 7                       # Polynomial order of the MW basis
  basis_type = Legendre           # Polynomial type (Legendre or Interpolating)
  min_scale = -4                  # Root scale defines world size 2^{-n}
  max_scale = 25                  # Maximum refinement 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
$coords
H   0.0     0.0    -0.7
H   0.0     0.0     0.7
$end
}

WaveFunction {
  method = HF                     # Wave function method (HF or DFT)
}

Properties {
  scf_energy = true               # Compute ground state energy
}

SCF {
  orbital_thrs = 1.0e-3           # Convergence threshold in orbital residual
  property_thrs = 1.0e-6          # Convergence threshold in total energy
  initial_guess = GTO             # Type of initial guess: none, mw, gto
  kain = 3                        # Length of KAIN iterative history
}

