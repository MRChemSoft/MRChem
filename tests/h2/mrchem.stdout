


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION X.Y.Z (rev. 5786fef)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Jonas Juselius   <jonas.juselius@uit.no>          ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : eb0dc8c

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 7
 polynomial type  = Legendre
------------------------------------------------------------
 unit length      = 16.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -16.000000  -16.000000  -16.000000 ]
 upper bounds     = [   16.000000   16.000000   16.000000 ]
 total length     = [   32.000000   32.000000   32.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00000e-01
  2     H           0.00000e+00   0.00000e+00   7.00000e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     H            1.00000e+00   1.00000e-04   7.57698e-03
  2     H            1.00000e+00   1.00000e-04   7.57698e-03
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    p    2                       9.999999999923315e-01
------------------------------------------------------------
                 Wall time: 2.95756e-01 sec 
============================================================


============================================================
 OrbitalVector:   1 orbitals     1 occupied     2 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0    -1       9.999999999923e-01    p   2    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.35658e-01
 Coulomb density                       328      1.62315e-01
 Coulomb potential                     136      1.67971e+00
 Coulomb grid size                     328          1343488
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.02463e+00
------------------------------------------------------------
                 Wall time: 4.20249e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        7.24349e-04
 Diagonalizing matrix                           2.12200e-06
 Rotating orbitals                              5.10041e-03
------------------------------------------------------------
                 Wall time: 5.85272e-03 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.842581100509648e+00
------------------------------------------------------------
 Total energy                        -1.128295386223934e+00
------------------------------------------------------------
                 Wall time: 3.02672e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.95938e-01
------------------------------------------------------------
                 Wall time: 1.36054e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 3.12116e-01
 Matrix part                                    1.59900e-06
------------------------------------------------------------
                 Wall time: 3.21157e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.94189e-01   328    0.00000e+00     0   6.15223e+00
------------------------------------------------------------
                 Wall time: 6.15225e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             9.40000e-07
 Copy orbitals                                  4.80389e-03
 Copy orbital updates                           5.96332e-03
------------------------------------------------------------
                 Wall time: 1.08181e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.47286e-01
 Coulomb density                       392      1.84329e-01
 Coulomb potential                     136      1.81588e+00
 Coulomb grid size                     392          1605632
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.10428e+00
------------------------------------------------------------
                 Wall time: 4.45201e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.96458e-01   1.82518e-02        328    p    2    0
------------------------------------------------------------
 Total error:                            1.82518e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -1.191875037873361e+00    -1.19188e+00    0
 Kinetic       1.073481349419756e+00     1.07348e+00    0
 N-E          -3.566768512565692e+00    -3.56677e+00    0
 Coulomb       1.301412125292535e+00     1.30141e+00    0
 Exchange     -6.507060626562469e-01    -6.50706e-01    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.842581100509648e+00    -1.84258e+00    0
 Nuclear       7.142857142857143e-01     7.14286e-01    0
------------------------------------------------------------
 Total        -1.128295386223934e+00    -1.12830e+00    0
============================================================


################ Wall time: 1.22892e+01 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        8.73722e-04
 Diagonalizing matrix                           1.50600e-06
 Rotating orbitals                              8.95628e-03
------------------------------------------------------------
                 Wall time: 9.85666e-03 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847562260445495e+00
------------------------------------------------------------
 Total energy                        -1.133276546159780e+00
------------------------------------------------------------
                 Wall time: 3.39511e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.96458e-01
------------------------------------------------------------
                 Wall time: 1.30314e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 3.30027e-01
 Matrix part                                    1.53800e-06
------------------------------------------------------------
                 Wall time: 3.38626e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.96920e-01   328    0.00000e+00     0   6.77725e+00
------------------------------------------------------------
                 Wall time: 6.77728e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             6.16000e-07
 Copy orbitals                                  6.94548e-03
 Copy orbital updates                           6.45335e-03
------------------------------------------------------------
                 Wall time: 1.34568e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.89466e-01
 Coulomb density                       392      1.81987e-01
 Coulomb potential                     136      1.90986e+00
 Coulomb grid size                     392          1605632
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.20349e+00
------------------------------------------------------------
                 Wall time: 4.68507e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.95362e-01   5.13506e-03        328    p    2    0
------------------------------------------------------------
 Total error:                            5.13506e-03    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -1.192915393566144e+00    -1.04036e-03    0
 Kinetic       1.104183913030674e+00     3.07026e-02    0
 N-E          -3.606393040355519e+00    -3.96245e-02    0
 Coulomb       1.309293733758702e+00     7.88161e-03    0
 Exchange     -6.546468668793513e-01    -3.94080e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847562260445495e+00    -4.98116e-03    0
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133276546159780e+00    -4.98116e-03    0
============================================================


################ Wall time: 1.30823e+01 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847882529337670e+00
------------------------------------------------------------
 Total energy                        -1.133596815051956e+00
------------------------------------------------------------
                 Wall time: 3.09658e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.95362e-01
------------------------------------------------------------
                 Wall time: 1.21369e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 3.30975e-01
 Matrix part                                    1.29100e-06
------------------------------------------------------------
                 Wall time: 3.40092e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.98928e-01   328    0.00000e+00     0   6.59555e+00
------------------------------------------------------------
                 Wall time: 6.59557e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             4.55000e-07
 Setup linear system                            1.65982e-02
 Solve linear system                            2.18900e-06
 Expand solution                                3.01571e-02
------------------------------------------------------------
                 Wall time: 4.68126e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.57155e-01
 Coulomb density                       392      1.80963e-01
 Coulomb potential                     136      1.74972e+00
 Coulomb grid size                     392          1605632
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.08140e+00
------------------------------------------------------------
                 Wall time: 4.36948e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.94691e-01   2.36447e-03        328    p    2    0
------------------------------------------------------------
 Total error:                            2.36447e-03    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -1.190724898872106e+00     2.19049e-03    0
 Kinetic       1.118667936112605e+00     1.44840e-02    0
 N-E          -3.623708095915838e+00    -1.73151e-02    0
 Coulomb       1.314315260931124e+00     5.02153e-03    0
 Exchange     -6.571576304655611e-01    -2.51076e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847882529337670e+00    -3.20269e-04    0
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133596815051956e+00    -3.20269e-04    0
============================================================


################ Wall time: 1.25593e+01 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847913482925915e+00
------------------------------------------------------------
 Total energy                        -1.133627768640201e+00
------------------------------------------------------------
                 Wall time: 3.09720e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.94691e-01
------------------------------------------------------------
                 Wall time: 1.16240e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 3.30356e-01
 Matrix part                                    1.43600e-06
------------------------------------------------------------
                 Wall time: 3.40098e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.99981e-01   328    0.00000e+00     0   6.12894e+00
------------------------------------------------------------
                 Wall time: 6.12896e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.11196e-03
 Setup linear system                            5.37432e-02
 Solve linear system                            4.04700e-06
 Expand solution                                5.28179e-02
------------------------------------------------------------
                 Wall time: 1.07734e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.52075e-01
 Coulomb density                       392      1.71025e-01
 Coulomb potential                     136      1.69138e+00
 Coulomb grid size                     392          1605632
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.03892e+00
------------------------------------------------------------
                 Wall time: 4.25362e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.94667e-01   1.25504e-04        328    p    2    1
------------------------------------------------------------
 Total error:                            1.25504e-04    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -1.189382037942233e+00     1.34286e-03    0
 Kinetic       1.125911302163791e+00     7.24337e-03    0
 N-E          -3.632356230073388e+00    -8.64813e-03    0
 Coulomb       1.317062889967365e+00     2.74763e-03    0
 Exchange     -6.585314449836827e-01    -1.37381e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847913482925915e+00    -3.09536e-05    0
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133627768640201e+00    -3.09536e-05    0
============================================================


################ Wall time: 1.20167e+01 sec ################




####################### SCF cycle  5 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847913551046316e+00
------------------------------------------------------------
 Total energy                        -1.133627836760601e+00
------------------------------------------------------------
                 Wall time: 3.02627e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -5.94667e-01
------------------------------------------------------------
                 Wall time: 1.15214e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 3.14174e-01
 Matrix part                                    1.54500e-06
------------------------------------------------------------
                 Wall time: 3.24190e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.99999e-01   328    0.00000e+00     0   6.02903e+00
------------------------------------------------------------
                 Wall time: 6.02905e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.29847e-03
 Setup linear system                            4.82856e-02
 Solve linear system                            4.60000e-06
 Expand solution                                4.43694e-02
------------------------------------------------------------
                 Wall time: 9.40149e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.39654e-01
 Coulomb density                       392      1.73908e-01
 Coulomb potential                     136      1.72499e+00
 Coulomb grid size                     392          1605632
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.06561e+00
------------------------------------------------------------
                 Wall time: 4.30441e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.94664e-01   1.32135e-05        328    p    2    1
------------------------------------------------------------
 Total error:                            1.32135e-05    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -1.189334355541795e+00     4.76824e-05    0
 Kinetic       1.126025506145426e+00     1.14204e-04    0
 N-E          -3.632518252696262e+00    -1.62023e-04    0
 Coulomb       1.317158391009043e+00     9.55010e-05    0
 Exchange     -6.585791955045219e-01    -4.77505e-05    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847913551046316e+00    -6.81204e-08    1
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133627836760601e+00    -6.81204e-08    1
============================================================


################ Wall time: 1.19426e+01 sec ################


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        8.80923e-04
 Diagonalizing matrix                           1.62500e-06
 Rotating orbitals                              7.70654e-03
------------------------------------------------------------
                 Wall time: 8.61753e-03 sec 
============================================================


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    1.82518e-02    -1.128295386223934e+00   -1.12830e+00
  2    5.13506e-03    -1.133276546159780e+00   -4.98116e-03
  3    2.36447e-03    -1.133596815051956e+00   -3.20269e-04
  4    1.25504e-04    -1.133627768640201e+00   -3.09536e-05
  5    1.32135e-05    -1.133627836760601e+00   -6.81204e-08
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     776      3.45059e-01
 Coulomb density                       392      1.73903e-01
 Coulomb potential                     136      1.72602e+00
 Coulomb grid size                     392          1605632
 Coulomb grid change                     0                0
 Hartree-Fock exchange                 328      2.05410e+00
------------------------------------------------------------
                 Wall time: 4.29930e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 3.51092e-01 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-5.94664e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    p    2                      -5.946637397325155e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00000e-01
  2     H           0.00000e+00   0.00000e+00   7.00000e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -1.189327479465031e+00
 Kinetic energy:                      1.126054554647350e+00
 E-N energy:                         -3.632554178632285e+00
 Coulomb energy:                      1.317172144520599e+00
 Exchange energy:                    -6.585860722606467e-01
 X-C energy:                          0.000000000000000e+00
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy                   -1.847913551724983e+00
 Nuclear energy                       7.142857142857143e-01
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -1.133627837439269e+00
                    (kJ/mol)         -2.976339456418222e+03
                    (kcal/mol)       -7.113622023151338e+02
                    (eV)             -3.084758358795875e+01
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 7.180171e+01                ***
***                                                      ***
************************************************************


